### ISSUE TITLE
Collision is done between 2 convex hulls ?

### ISSUE THREAD

**Issue by jiapei-nexera** [yuehaowang/convex_hull_3d#3]
What about concave hulls? For our application, we do prefer the collision is detected between two **TIGHT** 3D meshes, rather than the **CONVEX HULL** of two 3D objects.

I mean: if you can do two **CONCAVE HULL**s ?

Any suggestions?

**Comment by jiapei100** [comment]
Can we increase the number of chunks to describe **CONCAVE HULL** ???

**Comment by jiapei100** [comment]
Yeah.... `__init()__` in [https://github.com/yuehaowang/convex_hull_3d/blob/master/collision_detection.py](https://github.com/yuehaowang/convex_hull_3d/blob/master/collision_detection.py) shows everything ....

How can we deal with the collision between 2 **concave hulls** ??

**Comment by yuehaowang** [comment]
The underlying algorithm of collision detection in this repo is SAT, which only supports convex objects. A widely-used approach for handling non-convex objects is to first decompose non-convex objects into convex parts and then check intersection between those convex parts. I think you may refer to V-HACD (see [this repo](https://github.com/Unity-Technologies/VHACD)) for more details.
